components/http_foundation.rst

```diff
@@ -420,12 +420,12 @@ conditional value specified in the client Request, use the
 :method:`Symfony\\Component\\HttpFoundation\\Response::isNotModified`
 method::
 
-    if ($response->isNotModified($request)) {
+    if (!$response->isNotModified($request)) {
```

Comment:
But, the response's content should *already* be populated, long before prepare/isNotModified/send are called. This isn't a case of doing some tests to see if the response should be generated. By this point, the response has already been generated, and whatever is responsible for emitting the response is now in charge.

This isn't a case of the controller looking at the request to determine if what response is needed. The controller has already done that and returned a response it is satisfied with.